go-controller/pkg/ovn/egressip.go

```diff
@@ -760,6 +760,13 @@ type egressNode struct {
 	name               string
 }
 
+type allocator struct {
+	*sync.Mutex
```

Comment:
is this intentional to use a pointer to the mutex? If so what's the reason?

Reply:
It was intentional and there is no reason or benefit to use a pointer here. I will remove.